application/btsieve/config/default.toml

```diff
@@ -11,4 +11,3 @@ poll_interval_secs = 17
 [http_api]
 address_bind="0.0.0.0"
 port_bind=8080
-external_url="http://localhost:8080/"
```

Comment:
❌  similar clean up needed in regtest folder

Reply:
Good catch!